apps/web-giddh/src/app/vouchers/template/all-templates/templates/thermal-template/thermal-template.component.ts

```
@@ -0,0 +1,178 @@
+        this.invoiceUiDataService.customTemplate.pipe(takeUntil(this.destroyed$)).subscribe((template: CustomTemplateResponse) => {
+            if (template && template.sections) {
+                if (template.sections.footer.data.imageSignature?.display) {
+                    if (template.sections.footer.data.imageSignature.label) {
+                        this.imageSignatureSrc = ApiUrl + 'company/' + this.companyUniqueName + '/image/' + template.sections.footer.data.imageSignature.label;
+                    } else {
+                        this.imageSignatureSrc = '';
+                    }
+                } else {
+                    this.imageSignatureSrc = '';
+                }
+            } else if (template && template.sections && template.sections.footer.data.slogan?.display) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)